pkgs/development/compilers/llvm/rocm/llvm.nix

```diff
@@ -24,6 +24,8 @@
 , targetDir ? "llvm"
 , targetProjects ? [ ]
 , targetRuntimes ? [ ]
+# "NATIVE" resolves into x86 or aarch64 depending on stdenv
+, llvmTargetsToBuild ? [ "NATIVE" ]
```

Comment:
CC @NixOS/rocm-maintainers 

I merely needed a way to add NVPTX target when overriding llvm for triton. I'm happy to change the interface to whatever